index.html

```diff
@@ -1913,6 +1919,64 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    [=People=] should be provided relevant information when data is accessed or when permission is
+    requested.
+  </span>
+</div>
+
+Transparency is a necessary, but insufficient, condition for [=consent=]. Relevant information
+includes who may be accessing information, what information may be accessed (including the potential
+inferences or combinations of such data) and how information may be used. For transparency to be
+meaningful to people, it must be contextual.
```

Comment:
What does it mean for transparency to be contextual?

Reply:
Made it clear that it's the information that would be contextual, and instead described it as presenting the information in the relevant context (since context is a defined term). But this could still be clarified. And it may not be necessary for this document, even if it seems like generally good advice.